src/psyclone/tests/psyGen_test.py

```diff
@@ -2366,15 +2374,17 @@ def test_node_dag(tmpdir, have_graphviz):
     schedule.dag(file_name=my_file.strpath)
     result = my_file.read()
     print(result)
```

Comment:
Please remove all three of these print statements.

Reply:
Done